util/kubernetes_sa_init.py

```
@@ -0,0 +1,212 @@
+        superuser_subjects = service.superuser_subjects
+        for superuser_subject in superuser_subjects:
+            parsed = service.parse_sa_subject(superuser_subject)
+            if not parsed:
+                logger.warning(f"Invalid SUPERUSER_SUBJECTS entry: {superuser_subject}")
+                continue
+
+            namespace, sa_name = parsed
+            robot_shortname = service.generate_robot_shortname(namespace, sa_name)
+
+            logger.info(f"Initializing superuser robot for: {superuser_subject}")
+            robot = ensure_kubernetes_sa_robot(
```

Ruff (0.15.4)

[error] Local variable `robot` is assigned to but never used

Remove assignment to unused variable `robot`

(F841)